### PR TITLE
Refactor work type

### PR DESCRIPTION
**Pull request by GustavoZHA**

### CODE REVIEW

File: src/main/java/core/selenium/WebElementAction.java [hunk 1]
```diff
@@ -133,11 +134,11 @@ public WebElement getElement(final By locator) {
     /**
      * Selects a webElement isn't able to click.
      *
-     * @param webElement type WebElement object.
+     * @param by type WebElement object.
      */
-    public void selectByAction(final WebElement webElement) {
+    public void selectByAction(final By by) {
```
Comment (JorgeCaceres-Jala) [inline src/main/java/core/selenium/WebElementAction.java:139]: by is not a very descriptive name

**Review by JorgeCaceres-Jala — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -24,6 +24,8 @@ dependencies {
     implementation group: 'io.cucumber', name: 'cucumber-java', version: '6.10.4'
     implementation group: 'io.cucumber', name: 'cucumber-testng', version: '6.10.4'
     implementation group: 'io.cucumber', name: 'cucumber-picocontainer', version: '6.10.4'
+    implementation group: 'log4j', name: 'log4j', version: '1.2.17'
+
```
Comment (cristianfchq) [inline build.gradle:28]: this line should be removed

File: src/main/java/salesforce/ui/pages/lightning/worktype/NewWorkTypePage.java [hunk 3]
```diff
@@ -21,57 +20,65 @@
...
+    private String xpathComboBoxSomeFields = "//*[contains(text(),'%s')]/../..//a[@class='select']";
+    private String xpathValueComboBoxSomeFields = "//div[@aria-labelledby][@id][%d]//*[contains(text(),'%s')]";
+    private int countComboBox = 0;
 
     @Override
     protected void waitForPageLoaded() {
         webElementAction.waitForVisibilityOfLocator(estimatedDurationComboBox);
     }
 
+
```
Comment (cristianfchq) [inline src/main/java/salesforce/ui/pages/lightning/worktype/NewWorkTypePage.java:37]: this line should be removed

File: src/main/java/salesforce/ui/pages/lightning/worktype/NewWorkTypePage.java [hunk 4]
```diff
@@ -21,57 +20,65 @@
...
-        }
-        webElementAction.clickByXpath("//div[@class='actionsContainer']");
+        webElementAction.dropDownTillTheEnd();
+        webElementAction.setInputField(By.xpath(String.format(selectFieldTxtBox, fieldName)), fieldValue);
     }
 
     /**
      * Sets value and select estimated duration comboBox.
      *
+     *
```
Comment (cristianfchq) [inline src/main/java/salesforce/ui/pages/lightning/worktype/NewWorkTypePage.java:62]: this line should be removed

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**